.github/renos_updated.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# This script makes sure a reno has been updated in the PR.
+
+reno lint
+
```

Comment:
```suggestion
```
See suggestion to always run `reno lint` and just put it directly in the job steps.

Reply:
Completed in 737697a